tools/internal/ios/ios.webpack.config.js

```diff
@@ -4,13 +4,16 @@
 *--------------------------------------------------------------------------------------------*/
 
 const path = require("path");
-const glob = require("glob");
+const { globSync } = require("glob");
 
 module.exports = {
   mode: "development",
   entry: [
     path.resolve(__dirname, "scripts/configureMocha.js"),
-    ...glob.sync(path.resolve(__dirname, path.relative(__dirname, process.env.TESTS_GLOB))),
+    ...globSync(
+      path.resolve(__dirname, path.relative(__dirname, process.env.TESTS_GLOB)),
+      { windowsPathsNoEscape: true }
```

Comment:
not required, as this should only be used on macs, but its ok to have

Reply:
Removed the `windowsPathsNoEscape` option from the internal iOS tool